lib/mods/replace-constructors.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
+const nodeTypes = require('ts-morph').TypeGuards;
+const globalsSet = require('../globals-set.js');
+
 /**
- * Checks if the given constructor name is global
+ * @param {Node} node - a typed AST Node
  *
- * @param {String} constructor - the name of the constructor
+ * @return {Boolean} whether the type of the node is from TypeScript definitions
+ */
+function nodeInTSDefs(node) {
```

Comment:
Having these as separate functions is very nicely factored.

Reply:
Thanks!